example/src/main/java/io/netty/example/http2/helloworld/server/HelloWorldHttp2HandlerBuilder.java

```diff
@@ -31,6 +31,9 @@
 
     public HelloWorldHttp2HandlerBuilder() {
         frameLogger(logger);
+        // Running h2spec 5.5. that will send a large list without checking our max header list size
+        // so we need to set max header list size to pass the test
+        initialSettings().maxHeaderListSize(Integer.MAX_VALUE);
```

Comment:
we are sending a GO_AWAY which will tear down the connection ... so it is fine to tear down state.